test/printf.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Rrright.
We only generate i64 stores in printf when using the llvm wasm backend. Is there a way to convince asm2wasm to do so as part of the i64 wasm-only additions you added recently? If not, should I move this under the wasm_backend tests?